textworld/generator/game.py

```diff
@@ -119,7 +121,7 @@ def set_conditions(self, conditions: Iterable[Proposition]) -> Action:
         return self.condition
 
     def __hash__(self) -> int:
-        return hash((self.actions,
+        return hash((tuple(self.actions),
```

Comment:
I originally left the tuple call there in case someone changes `Event.actions` to a list. I don't think that should ever happen (not that we prevent it though) but it does happen for `Event..commands` hence the tuple call on the next line. I can remove it if you think it is a problem.

Reply:
Ah okay I didn't see the assignments to `.commands`.  Looks fine then.